recaf-assembler/src/main/java/me/coley/recaf/assemble/transformer/AnnotationHelper.java

```diff
@@ -30,7 +31,7 @@ public class AnnotationHelper {
 	 * @param annotations
 	 * 		The annotations to add.
 	 */
-	public static void visitAnnos(AbstractDefinition code, boolean visible, List<AnnotationNode> annotations) {
+	public static void visitAnnos(AbstractAttributable code, boolean visible, List<AnnotationNode> annotations) {
```

Comment:
If the purpose of `Attributable` is to say _"this type has annotations"_ then it should be `Annotatable` or something along those lines.

Reply:
The reason for `Attributable`s existance is that there exists a overlap for attributes with `declaration` and `record component` infos according to the [4.7-C Table](https://docs.oracle.com/javase/specs/jvms/se19/html/jvms-4.html#jvms-4.7.20) in the jvm. Now because the signature and annotation fields and spec are trapped within the `Definition` interface i needed to extract it to a `Attributable` (name i came up with at first (dont know if correct)) interface which then interfaces `AbstractAttributable` which is extended by `AbstractDefinition` AND `RecordComponent` and because the method only supported reading annotations into a `AbstractDefinition`  i needed to make it adapt to the attributable and i dont name it `Annotable` because `Attributable` contains more than one attribute in it. While i do acknowledge creating a interface spec for 2 fields can be considered excessive but creating 2 extra interfaces for them is even more